packages/graph/hash_graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -77,10 +77,11 @@ impl Transpile for Condition<'_> {
                 fmt.write_str(" != ")?;
                 rhs.transpile(fmt)
             }
-            Condition::RangeContains(lhs, rhs) => {
+            Condition::TimerangeContainsTimestamp(lhs, rhs) => {
                 lhs.transpile(fmt)?;
                 fmt.write_str(" @> ")?;
-                rhs.transpile(fmt)
+                rhs.transpile(fmt)?;
+                fmt.write_str("::TIMESTAMPTZ")
```

Comment:
Somehow, `tokio_postgres` tries to serialize a `TSTZRANGE` instead of `TIMESTAMPTZ`